apps/src/templates/feedback/shapes.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const shapes = {
```

Comment:
i love this convention! +1 for using this everywhere to keep prop definitions more centralized